train.py

```diff
@@ -0,0 +1,302 @@
+"""
+train.py
+Training script for the few-shot keyword detection model.
+"""
+
+import logging
+import os
+import subprocess
+from pathlib import Path
+
+import numpy as np
+import tensorflow as tf
+from sklearn.model_selection import train_test_split
+from typer import Typer, run
+
+from python_path import PythonPath
+
+with PythonPath("./"):
+    from multilingual_kws.data_pipeline import (
+        standard_microspeech_model_settings,
+        file2spec,
+    )
+    from multilingual_kws.embedding.transfer_learning import (
+        transfer_learn,
+    )
+
+app = Typer()
+
+# Set up logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+file_path = Path(__file__)
+data_dir = file_path.parent.parent.parent / "data/keyword_dataset"
+
+
+def main(
+    toy: bool,
+) -> None:
+    """
+    Main function for training the few-shot keyword detection model.
+
+    Args:
+        toy: Whether to use a toy dataset for training.
+    """
+    # Using subprocess to call ls command and save the output in list format
+
+    original_keyword_examples = (
+        subprocess.check_output(
+            [
+                "find",
+                os.path.join(data_dir, "ours", "original-keyword-examples"),
+                "-type",
+                "f",
+                "-name",
+                "*.wav",
+            ]
+        )
+        .decode("utf-8")
+        .split("\n")
+    )
+    crawling_keyword_examples = (
+        subprocess.check_output(
+            [
+                "find",
+                os.path.join(data_dir, "ours", "crawling-keyword-examples"),
+                "-type",
+                "f",
+                "-name",
+                "*.wav",
```

Comment:
add comment of what that data is about

Reply:
But these data is in BigQuery, why do you need to use gsutil list?

Reply:
The ones that are on this directory are not all the images that we recorded; first I took max 15 per person, and also It turns out that all the recordings that we did are not in the correct format, so I used the script I left on Tamagotchi repo tu convert them

Reply:
Done

Reply:
-.-